plugins/UM3NetworkPrinting/resources/qml/MonitorContextMenuButton.qml

```diff
@@ -18,7 +18,7 @@ Button
         width: base.width
     }
     contentItem: Label {
-        color: enabled ? UM.Theme.getColor("monitor_text_primary") : UM.Theme.getColor("monitor_text_disabled")
+        color: enabled ? UM.Theme.getColor("text") : UM.Theme.getColor("monitor_text_disabled")
```

Comment:
You probably want to use an alternative for monitor_text_disabled too then.

Reply:
Yeah, I started this in order to find some quick replacements, but unfortunately, we don't really have disabled text quite the same way in other parts. 

I will spend some extra time on this. The sheer amount of TODO's in this code is just painfull.